Makefile

```diff
@@ -189,6 +189,12 @@ HUGO ?= $(LOCALBIN)/hugo-$(HUGO_VERSION)
 export KUBE_VERSION ?= 1.25
 KIND_CONFIG ?= kind-$(KUBE_VERSION).yaml
 
+# Choose wich version to generate
+BUNDLE_VARIANT ?= community
```

Comment:
How is this use? I don't see where it's getting overridden to openshift.

Reply:
Is not used in this PR, or not yet

the idea is to have `BUNDLE_VARIANT=openshift make bundle`

Reply:
I would maybe prefer to remove it for now if there is no use-case to generate just a single bundle.

Reply:
Now I generate both bundles with `make bundle`. So it is used there.